### PR TITLE
Java batch update

### PR DESCRIPTION
**Pull request by andreamlin**

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
It looks like the build is failing because bigtable now needs iam.

**Comment by garrettjonesgoogle** [comment]
Also you seem to be including class files, those should be removed.

File: generated/java/grpc-google-longrunning-v1/src/main/java/com/google/longrunning/OperationsGrpc.java [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+ * operation resource, or pass the operation resource to another API (such as
+ * Google Cloud Pub/Sub API) to receive the response.  Any API service that
+ * returns long-running operations should implement the `Operations` interface
+ * so developers can have a consistent client experience.
+ * </pre>
+ */
+@javax.annotation.Generated(
+    value = "by gRPC proto compiler (version 1.2.0)",
+    comments = "Source: google/longrunning/operations.proto")
+public final class OperationsGrpc {
```
Comment (garrettjonesgoogle) [inline generated/java/grpc-google-longrunning-v1/src/main/java/com/google/longrunning/OperationsGrpc.java]: Ok there is an issue - this shouldn't be added back. I think this is a mess I left behind. Basically, proto and grpc classes for longrunning are included in common-protos; we only want a gapic folder for longrunning.

**Comment by andreamlin** [comment]
Removed class files, and fixed iam dependency in bigtable.

File: generated/java/grpc-google-longrunning-v1/src/main/java/com/google/longrunning/OperationsGrpc.java (hunk 1 above)
Comment (andreamlin) [reply generated/java/grpc-google-longrunning-v1/src/main/java/com/google/longrunning/OperationsGrpc.java]: ok, removed proto-google-longrunning-v1 and grpc-google-longrunning-v1

Comment (andreamlin) [reply generated/java/grpc-google-longrunning-v1/src/main/java/com/google/longrunning/OperationsGrpc.java]: and replaced the longrunning gapic's dependency on those classes with the common-protos

**Comment by andreamlin** [comment]
PTAL
Not sure why the openjdk7 build fails on a STANDARD_ERROR that passes in tests outside of the logging API; openjdk8 build does pass!

**Pull request merged**